README.md

```diff
@@ -63,16 +75,68 @@ By default, when the provider triggers a reboot, it will skip any resources in t
       apply  => finished,
     }
 
-##Limitations
+##Reference
+
+###Type: reboot
+
+The main type of the module, responsible for all its functionality.
+
+####Providers
+
+* `windows`: Default for :kernel => :windows
+* `linux`: Default for :kernel => :linux
+
+####Features
+
+* `supports_reboot_prompting`: Notifies the node's user that a reboot is pending, and let them cancel the pending reboot process. (Available with the `windows` provider. Linux terminal users get a shutdown notice regardless of this parameter, but they can't cancel the reboot.)
```

Comment:
@elyscape Do you have an idea of when #62 will be merged? Happy to update my PR, if it's imminent—or I can do another pass on the README once it does go through.